src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -107,13 +107,13 @@ public static IHostBuilder AddScriptHostCore(this IHostBuilder builder, ScriptAp
 
                 var extensionBundleOptions = GetExtensionBundleOptions(context);
                 var bundleManager = new ExtensionBundleManager(extensionBundleOptions, SystemEnvironment.Instance, loggerFactory);
+                webJobsBuilder.Services.AddSingleton<IExtensionBundleManager>(_ => bundleManager);
```

Comment:
Decided to move this anyway?

Reply:
yeah, wanted to increase the chance of service resolution.